content/blog/2024/leap-pangeo-paper/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Researchers at LEAP-Pangeo investigate overlooked sub-grid air-sea heat flux in climate models
+date: "2024-06-19"
```

Comment:
The [post](https://2i2c.org/blog/2024/leap-pangeo-paper/) somehow shows May as publish date? Not sure how that happens, but just noticed it.

Reply:
Ah yes we are "backdating" some posts as we are writing up quite a lot of items at once rn 😅